docs/doc/code-review.md

```diff
@@ -3,15 +3,22 @@ id: code-review
 title: How to Conduct A Helpful Code Review
 ---
 
-Consider following links:
+As a reviewer consider the following guidelines:
 
 - [How to Do Code Reviews Like a Human (Part One)](https://mtlynch.io/human-code-reviews-1/) /
   [Code review по-человечески (часть 1)](https://habr.com/en/post/340550/)
```

Comment:
Maybe remove `habr` links? Russian site, doesn't work without VPN.

Reply:
Makes sense as we have links to original posts anyway and code reviewers are likely have sufficient command of English.
Cannot remember who suggested adding translations and why. Would expect yet another opinion on this matter.